pkg/targetgroupbinding/resource_manager.go

```diff
@@ -539,3 +549,17 @@ func isELBV2TargetGroupARNInvalidError(err error) bool {
 	}
 	return false
 }
+
+func parseCIDR(tgb *elbv2api.TargetGroupBinding) ([]netip.Prefix, error) {
+	var cidrs []netip.Prefix
+	// only triggered if the object has an annotation named `annotation`
+	if annotation := tgb.Annotations["cidrs"]; annotation != "" {
```

Comment:
update to something a lil more saucy

Reply:
`mega-target-group-cidrs` no one can stop me